packages/notification-services-controller/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tsconfig.packages.build.json",
+  "extends": "../../tsconfig.packages.json",
```

Comment:
Misc, noticed we are not using the correct tsconfig for this file compared to other packages. No major issues though.